internal/endtoend/testdata/mix/test.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE bar (
+	id serial not null,
+	name text not null,
+	phone text not null
+);
+
+-- name: CountOne :one
+SELECT count(1) FROM bar WHERE id = $2 AND phone < @phone_param and name <> $1;
+
+-- name: CountTwo :one
+SELECT count(1) FROM bar WHERE id = sqlc.arg(id_param) AND phone < @phone_param and name <> $1;
```

Comment:
The @param style will be deprecated soon (see #605) so let's remove them from the test.